python/pybullet/grasping_demo/pr2_dual_push_scoop_grasp_demo.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import sys
+from math import *
+import pybullet 
+import pybullet as pb
+import pybullet_data
+import numpy as np
+import time
+import six
+import utils
+from grasp_utils import *
+
+from gripper import RGripper, LGripper
+
+
+class Simulator(object):
+    def __init__(self):
+        try:
```

Comment:
this `try`-`except` does nothing. Because everytime you run the code in interpreter, you make the new `sim` object, and initialization procedures occurs for the new object, and error occurs because another pybullet CLIENT will be attempted to be made. (please try)

Rather, you should write like 
```python 
try:
    sim 
except:
    sim = Simulartor()
```

Reply:
Maybe got it. I change the code you've mentioned.